backend/controller/user.go

```diff
@@ -17,20 +17,39 @@ import (
 	"net/smtp"
 	"os"
 	"strconv"
+	"strings"
 	"time"
 )
 
-func GetAllUsers(w http.ResponseWriter, _ *http.Request) {
+func GetAllUsers(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	err, users := dbHelpers.GetAllTable(w, []models.User{})
-	if err != nil {
-		return
+	var users []models.User
+	// Get user ids from the query
+	userIdsParam := r.URL.Query().Get("users")
+	if userIdsParam == "" {
+		var err error
+		err, users = dbHelpers.GetAllTable(w, []models.User{})
+		if err != nil {
+			return
+		}
+	} else {
+		// Split the comma-separated string into individual IDs
+		userIds := make([]int, 0)
+		for _, id := range strings.Split(userIdsParam, ",") {
+			intId, err := strconv.Atoi(id)
```

Comment:
sorry I didnt realize ud paste it exactly, still need to make it uint, I think u can just wrap unit(intvalue) not 100p

Reply:
Atoi converts it to an int, why do we need it to be a uint?

Reply:
cuz all our ids are uints?

Reply:
if u look at the helper function that normally does this its always converting to uint